src/buffer_manager.ts

```diff
@@ -442,6 +443,15 @@ export class BufferManager implements Disposable {
         );
     }
 
+    // #247
+    private handleBufferModifiedSet({ buf, modified }: EventBusData<"BufModifiedSet">[0]) {
+        if (modified) return; // expected and we can't do anything about it
```

Comment:
can we go a tiny bit further and be a bit more precise, that would help a lot here. something like:

```suggestion
        if (modified) return; // Nvim buffer is always 'modified', but this is irrelevant to us.
```

would also be nice if we consistently used the `on*` prefix instead of randomly naming things `handle*()`, but that's out of scope for this PR I guess.

Reply:
`This was received because we modified the buffer; this is expected.`

Reply:
> would also be nice if we consistently used the on* prefix instead of randomly naming things handle*(), but that's out of scope for this PR I guess.

Yes, "on" is for events, "handle" is for actions. Actually, it doesn't matter; both are fine.

Reply:
> This was received because we modified the buffer; this is expected.

Nice!

Reply:
> Yes, "on" is for events, "handle" is for actions. Actually, it doesn't matter; both are fine.

"actions" are not really a well-defined concept. And I think this is more evidence that the "actions" concept is redundant.